### PR TITLE
[RPC] getblock: Added help text for chainwork value

### PR DESCRIPTION
**Pull request by instagibbs**
I took the text from bitcoin.org's description. @harding Is this originally from there?  

edit: Changed the wording

### CODE REVIEW

**Comment by harding** [comment]
@instagibbs yes; that description was written by me based upon my reading of the code.

**Comment by fanquake** [comment]
ACK

File: src/rpcblockchain.cpp [hunk 1]
```diff
@@ -384,6 +384,7 @@ UniValue getblock(const UniValue& params, bool fHelp)
             "  \"nonce\" : n,           (numeric) The nonce\n"
             "  \"bits\" : \"1d00ffff\", (string) The bits\n"
             "  \"difficulty\" : x.xxx,  (numeric) The difficulty\n"
+            "  \"chainwork\" : \"xxxx\",  (string) The estimated number of block header hashes checked from the genesis block to this block, in hexadecimal\n"
```
Comment (MarcoFalke) [inline src/rpcblockchain.cpp]: I don't understand this sentence. I like `(string) Expected number of hashes required to produce the current chain (in hex)` better.

Comment (instagibbs) [inline src/rpcblockchain.cpp]: What don't you understand about it specifically? Would:  

>(string) Expected number of hashes required to produce the chain up to this block (in hex)  

be better? As-is it's ambiguous to what "current chain" means.

Comment (MarcoFalke) [inline src/rpcblockchain.cpp]: > block header hashes

is not immediately clear what this means. Usually blocks with hashes that don't hit the diff are not considered valid, so they are not counted. Then "estimated" gives the hint that they are counted as well.

Comment (MarcoFalke) [inline src/rpcblockchain.cpp]: @instagibbs I like your suggestion as well.

**Comment by instagibbs** [comment]
Updated wording re: @MarcoFalke comments.

**Comment by MarcoFalke** [comment]
utACK 94bdd71f9b4768c9803ffd133aa7781b19bfa6f9

**Comment by laanwj** [comment]
utACK

**Pull request merged**